fmriprep/cli/parser.py

```diff
@@ -541,12 +534,18 @@ def _bids_filter(value):
 
 def parse_args(args=None, namespace=None):
     """Parse args and run further checks on the command line."""
+    import os
     import logging
     from niworkflows.utils.spaces import Reference, SpatialReferences
     from niworkflows.utils.misc import check_valid_fs_license
 
     parser = _build_parser()
     opts = parser.parse_args(args, namespace)
+
+    if os.getenv('FMRIPREP_NO_REUSE_CONFIG') is not None:
```

Comment:
I'm not 100% positive this would be the right thing to do:
```suggestion
    # IMPORTANT: For the --reports-only operation, the last UUID must be retrieved.
    if os.getenv('FMRIPREP_NO_REUSE_CONFIG') is not None or opts.reports_only is not None:
```
but I'm sure we want to do something like this with `--reports-only`. Probably this feature does not work anymore, but that's out of the scope of the PR. Let's try to minimize the impact of removing ``--run-uuid`` somehow.